services/web/server/src/simcore_service_webserver/exceptions_handlers.py

```diff
@@ -0,0 +1,90 @@
+import functools
+import logging
+from collections.abc import Iterable
+from typing import NamedTuple, TypeAlias
+
+from aiohttp import web
+from servicelib.aiohttp.typing_extension import Handler
+from servicelib.aiohttp.web_exceptions_extension import get_http_error_class_or_none
+from servicelib.logging_errors import create_troubleshotting_log_kwargs
+from servicelib.status_codes_utils import is_5xx_server_error
+
+_logger = logging.getLogger(__name__)
+
+
+class HttpErrorInfo(NamedTuple):
+    status_code: int
+    msg_template: str
+
+
+ExceptionToHttpErrorMap: TypeAlias = dict[type[BaseException], HttpErrorInfo]
+
+
+class _DefaultDict(dict):
+    def __missing__(self, key):
+        return f"'{key}=?'"
```

Comment:
I think this would just generate confusion. Is the value of te key `?`. Could you make it more obvious, something like below, to avoid confusion:
```suggestion
        return f"'{key} was not found'"
```

Reply:
This is a protection mechanism to avoid exceptions within the exception handlers. It is intentionally short because it composes with a larger name. The purpose here is to raise an alarm because if this happens, there is a bug and needs to be resolved. It is like the OEC, if you like. We need to trace them. That is the reason for having `=?`